### PR TITLE
doc: add board terminology section to board porting guide

### PR DESCRIPTION
**Pull request by tejlmand**
Add a terminology section to the board porting guide. This provides a single location where terminologies used for boards can be found and linked to.

### CODE REVIEW

File: doc/hardware/porting/board_porting.rst [hunk 1]
```diff
@@ -956,3 +956,23 @@ extension.
    Board extensions are not allowed in any module referenced in Zephyr's
    ``west.yml`` manifest file. Any board changes are required to be submitted to
    the main Zephyr repository.
+
+.. _board_terminology:
+
+Terminology
+***********
+
+- board: a physical piece of hardware or an emulated board.
```
Comment (carlescufi) [inline doc/hardware/porting/board_porting.rst:965]: Please add an example for each of the terms

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: doc/hardware/porting/board_porting.rst [hunk 2]
```diff
@@ -956,3 +956,23 @@ extension.
    Board extensions are not allowed in any module referenced in Zephyr's
    ``west.yml`` manifest file. Any board changes are required to be submitted to
    the main Zephyr repository.
+
+.. _board_terminology:
+
+Terminology
```
Comment (nashif) [inline doc/hardware/porting/board_porting.rst:962]: would be nicer if you used some formatting for this type of thing, so it is easy to read and find:

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#create-a-glossary

Comment (nashif) [reply doc/hardware/porting/board_porting.rst:962]: would be nice as a followup to have few diagrams showing all of this and the layering and how things relate and what type of "setups" are possible. a picture is worth 1000 words....

Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst:962]: > would be nicer if you used some formatting for this type of thing, so it is easy to read and find:
> 
> https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#create-a-glossary

Thanks for the link, will take a look at updating.
I just followed what we do here: https://docs.zephyrproject.org/latest/project/index.html#terminology

Perhaps long term we should try to have a uniform terminology style, fyi @kartben

Comment (nashif) [reply doc/hardware/porting/board_porting.rst:962]: now that you mention this, you should remove the "/" from identifier :)

Comment (nashif) [reply doc/hardware/porting/board_porting.rst:962]: and yes, we can start with a link to this table.

Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst:962]: > > would be nicer if you used some formatting for this type of thing, so it is easy to read and find:
> > https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#create-a-glossary
> 
> Thanks for the link, will take a look at updating.

Done

Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst:962]: > and yes, we can start with a link to this table.

added a reference to the section as I couldn't get referencing the table directly to work.

Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst:962]: > now that you mention this, you should remove the "/" from identifier :)

done

**Review by carlescufi — DISMISSED** [review]
LGTM, thank you!

File: doc/hardware/porting/board_porting.rst [hunk 3]
```diff
@@ -956,3 +956,77 @@ extension.
...
+
+.. glossary::
+
+    board
+      a physical piece of hardware or an emulated board.
+
+    board identifier
+      a name which identifies the exact SoC, CPU cluster, or board
+      variant for the current build. The board identifier is separated from the
+      board name by a ``/``. Similar the SoC is also separated from the CPU
```
Comment (ajarmouni-st) [inline doc/hardware/porting/board_porting.rst:973]:
```suggestion
      board name by a ``/``. Similarly, the SoC is also separated from the CPU
```

File: doc/hardware/porting/board_porting.rst [hunk 4]
```diff
@@ -956,3 +956,77 @@ extension.
...
+``cpuapp``, furthermore defines a non-secure board variant, ``ns``.
+
+The board with identifier like ``nrf5340dk/nrf5340/cpuapp`` can be read as:
+
+- ``nrf5340dk``: This part is before the first ``/`` and is the board name.
+- ``nrf5340/cpuapp``: is the board identifier, and can be further divided into
+
+  - ``nrf5340``: The SoC, which is ``nrf5340``
+  - ``cpuapp``: The CPU cluster ``cpuapp`` which for this SoC is a single CPU
+    core of Cortex-m33. The number of cores in a CPU cluster cannot be
```
Comment (ajarmouni-st) [inline doc/hardware/porting/board_porting.rst:1012]:
```suggestion
  - ``cpuapp``: The CPU cluster ``cpuapp``, which is a single Cortex-M33 CPU
    core for this SoC. The number of cores in a CPU cluster cannot be
```

File: doc/hardware/porting/board_porting.rst [hunk 5]
```diff
@@ -956,3 +956,77 @@ extension.
...
+
+The ``cpuapp`` also defines a non-secure board variant, which is identified as
+``nrf5340dk/nrf5340/cpuapp/ns``. The first part of the board identifier has been
+explained above, and the extra part is read as:
+
+- ``ns``: a board variant,, in this case ``ns`` is a common variant name is
+  Zephyr denoting a non-secure build for boards supporting :ref:`tfm`.
+
+Not all SoCs defines CPU clusters or board variants. For example a simple board
+like the :ref:`thingy52_nrf52832` contains a single SoC with no CPU cluster and
```
Comment (ajarmouni-st) [inline doc/hardware/porting/board_porting.rst:1023]:
```suggestion
Not all SoCs define CPU clusters or board variants. For example a simple board
like the :ref:`thingy52_nrf52832` contains a single SoC with no CPU clusters and
```

File: doc/hardware/porting/board_porting.rst [hunk 6]
```diff
@@ -956,3 +956,77 @@ extension.
...
+no board variants.
+For ``thingy52`` the board with identifier like ``thingy52/nrf52832`` can be
+read as:
+
+- ``thingy52``: This part is before the first ``/`` and is the board name.
+- ``nrf52832``: is the board identifier, and as there is no more levels then
+  it's in this case identical the SoC, which is ``nrf52832``.
```
Comment (ajarmouni-st) [inline doc/hardware/porting/board_porting.rst:1030]:
```suggestion
- ``nrf52832``: is the board identifier, and as there are no more levels then
  it's in this case identical to the SoC, which is ``nrf52832``.
```

File: doc/hardware/porting/board_porting.rst [hunk 7]
```diff
@@ -956,3 +956,77 @@ extension.
    Board extensions are not allowed in any module referenced in Zephyr's
    ``west.yml`` manifest file. Any board changes are required to be submitted to
    the main Zephyr repository.
+
+.. _board_terminology:
+
+Terminology
```
Comment (kartben) [inline doc/hardware/porting/board_porting.rst:962]: sorry if I missed something but this list is almost exactly a copy-paste of the "Boards, SoCs, etc." section, so I am not sure there is a reason for the two to still co-exist?

**Review by kartben — CHANGES_REQUESTED** [review]

File: doc/hardware/porting/board_porting.rst (hunk 7 above)
Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst:962]: there was a discussion on having a terminology section, see https://discord.com/channels/720317445772017664/1194671118422913114/1214257604205744158 and before comments. A proposal to which I agree, especially as it also makes it easier to reference.

But you're right that `Boards, SoCs, etc.` contains part of same, but the reason I didn't remove it from there is because I still think it makes sense to have a brief description at that location where we introduces the concepts.

At the same time, a reader will expect that a terminology section describes all important terms, as a form of quick reference. So hard to argue those term should not be there.

File: doc/hardware/porting/board_porting.rst [hunk 8]
```diff
@@ -956,3 +956,77 @@ extension.
...
+      variant for the current build. The board identifier is separated from the
+      board name by a ``/``. Similar the SoC is also separated from the CPU
+      cluster and variant by a ``/``.
+
+    board revision
+      a specific revision of a board when the board exists in multiple revisions
+      and where a given revision needs adjustments to devicetree or Kconfig
+      files for correct behavior.
+
+     SoC
```
[the remark this answers is not in the document]
Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst]: indentation fixed, and list sorted alphabetically.

[the remark this answers is not in the document]
Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst]: > the rendered page looks very funny:

Even though I have forced pushed with alphabetically sorted list and indentation fixed, the doc build output has not been updated.

**Review by carlescufi — DISMISSED** [review]

File: doc/hardware/porting/board_porting.rst (hunk 8 above)
[the remark this answers is not in the document]
Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst]: This is fixed now as far as I can tell:
https://builds.zephyrproject.io/zephyr/pr/69802/docs/hardware/porting/board_porting.html#terminology

File: doc/hardware/porting/board_porting.rst (hunk 7 above)
Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst:962]: I think we can live with this duplication. Longer term what we need is a linkable global Glossary instead, like we do in NCS:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/glossary.html

Comment (kartben) [reply doc/hardware/porting/board_porting.rst:962]: > I think we can live with this duplication. Longer term what we need is a linkable global Glossary instead, like we do in NCS:
> https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/glossary.html
> 

Exactly :) What's wrong with using the glossary we already have? The text that's used in "Boards, SoCs, ..." can stay there as a legend for the diagram, but should also link to proper entries in the glossary (and this will be a good opportunity to update existing entries such as "SoC")
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/glossary.rst

**Review by gmarull — DISMISSED** [review]

**Review by ajarmouni-st — DISMISSED** [review]

File: doc/hardware/porting/board_porting.rst [hunk 9]
```diff
@@ -157,19 +157,19 @@ not always used.
...
+   |                                            | nrf5340/cpunet        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`mimx8mp_evk <imx8mp_evk>`            | /mimx8m/a53           | i.MX8M Plus | i.MXM8M A53   | NXP i.MX      | Arm Cortex-A53 | Arm64        |
+   | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8m/a53            | i.MX8M Plus | i.MXM8M A53   | NXP i.MX      | Arm Cortex-A53 | Arm64        |
    |                                            +-----------------------+-------------+---------------+---------------+----------------+--------------+
-   |                                            | /mimx8m/m7            | i.MX8M Plus | i.MXM8MM M4   | NXP i.MX      | Arm Cortex-M7  | Arm          |
+   |                                            | mimx8m/m7             | i.MX8M Plus | i.MXM8MM M4   | NXP i.MX      | Arm Cortex-M7  | Arm          |
```
Comment (iuliana-prodan) [inline doc/hardware/porting/board_porting.rst:172]: The SoC Series is wrong here - it should be i.MX8M M7 - https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/nxp/imx/soc.yml#L32

Comment (iuliana-prodan) [reply doc/hardware/porting/board_porting.rst:172]: Also, I would add here another entry for other arch, like Xtensa:
HiFi4 | Xtensa |  mimx8m/adsp | i.MX8M Plus | i.MX8M ADSP | NXP i.MX | HIFi4

File: doc/hardware/porting/board_porting.rst [hunk 10]
```diff
@@ -157,19 +157,19 @@ not always used.
...
+   | :ref:`nrf5340dk <nrf5340dk_nrf5340>`       | nrf5340/cpuapp        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    |                                            +-----------------------+-------------+---------------+---------------+----------------+--------------+
-   |                                            | /nrf5340/cpunet       | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
+   |                                            | nrf5340/cpunet        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`mimx8mp_evk <imx8mp_evk>`            | /mimx8m/a53           | i.MX8M Plus | i.MXM8M A53   | NXP i.MX      | Arm Cortex-A53 | Arm64        |
+   | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8m/a53            | i.MX8M Plus | i.MXM8M A53   | NXP i.MX      | Arm Cortex-A53 | Arm64        |
```
Comment (iuliana-prodan) [inline doc/hardware/porting/board_porting.rst:170]: s/i.MXM8M A53/i.MX8M A53

**Review by iuliana-prodan — CHANGES_REQUESTED** [review]

File: doc/hardware/porting/board_porting.rst (hunk 10 above)
Comment (tejlmand) [reply doc/hardware/porting/board_porting.rst]: see https://github.com/zephyrproject-rtos/zephyr/pull/69802#discussion_r1514519675

File: doc/hardware/porting/board_porting.rst [hunk 11]
```diff
@@ -157,19 +157,19 @@ not always used.
...
+   | :ref:`nrf5340dk <nrf5340dk_nrf5340>`       | nrf5340/cpuapp        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    |                                            +-----------------------+-------------+---------------+---------------+----------------+--------------+
-   |                                            | /nrf5340/cpunet       | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
+   |                                            | nrf5340/cpunet        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`mimx8mp_evk <imx8mp_evk>`            | /mimx8m/a53           | i.MX8M Plus | i.MXM8M A53   | NXP i.MX      | Arm Cortex-A53 | Arm64        |
+   | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8m/a53            | i.MX8M Plus | i.MX8M        | NXP i.MX      | Arm Cortex-A53 | Arm64        |
```
Comment (iuliana-prodan) [inline doc/hardware/porting/board_porting.rst:170]: Is `mimx8m` the correct identifier?
In boards/ we have `mimx8ml8`.
See:
A53 -  https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_a53.yaml#L7
M7 - https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_m7.yaml#L7

Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst]: Done

**Review by iuliana-prodan — DISMISSED** [review]

File: doc/hardware/porting/board_porting.rst [hunk 12]
```diff
@@ -67,27 +67,27 @@ complete conversion reference.
 
 .. _board_and_identifiers:
 
-Board and board identifiers
-***************************
+Board names, identifiers and targets
```
Comment (nashif) [inline doc/hardware/porting/board_porting.rst:70]: qualifiers?

Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst:70]: Sorry, this is not ready for review yet. Working on it now

**Review by iuliana-prodan — DISMISSED** [review]

File: doc/glossary.rst [hunk 13]
```diff
@@ -85,12 +128,30 @@ Glossary of Terms
...
+      features, and that the vendor typically names and markets together.
 
    system power state
       System power states describe the power consumption of the system as a
       whole. System power states are represented by :c:enum:`pm_state`.
 
+   variant
+      In the context of :term:`board qualifiers`, a variant designates a
+      particular type or configuration of a build for a combination of :term:`SoC`
+      and :term:`CPU cluster`. Commons uses of the variant concept include
```
Comment (ajarmouni-st) [inline doc/glossary.rst:150]:
```suggestion
      and :term:`CPU cluster`. Common uses of the variant concept include
```

**Review by ajarmouni-st — DISMISSED** [review]

**Review by ajarmouni-st — DISMISSED** [review]

File: doc/hardware/porting/board_porting.rst [hunk 14]
```diff
@@ -65,112 +65,128 @@ complete conversion reference.
...
+Zephyr's hardware support hierarchy has the following levels, from most to least
+specific:
 
-It's possible to define variants for dedicated use-cases.
-Examples of such use-cases are:
+- :term:`board`, which has one or more
+- :term:`SoC`, each of which optionally belong to a
+- :term:`SoC series`, which in turn may optionally belong to a
+- :term:`SoC family`, which has one or more
+- :term:`CPU Cluster`, each containing one or more
```
Comment (tejlmand) [inline doc/hardware/porting/board_porting.rst:80]: minor nit, this could give the impression that it is the SoC family which defines / has one or more CPU cluster.

In fact it is the SoC which defines the CPU cluster, not the family.

File: doc/hardware/porting/board_porting.rst [hunk 15]
```diff
@@ -65,112 +65,128 @@ complete conversion reference.
...
-.. _hw_support_hierarchy:
+which can be extended to:
 
-Boards, SoCs, etc.
-******************
+.. code-block:: console
 
-Zephyr's hardware support hierarchy has these layers, from most to least
-specific:
+   {board name}[@{revision}][/{SoC}][/{CPU cluster}][/{variant}]
```
Comment (tejlmand) [inline doc/hardware/porting/board_porting.rst:159]: to me this indicates that you can have a CPU cluster or variant without a SoC.

Should this have been:
```suggestion
   {board name}[@{revision}][/{SoC}[/{CPU cluster}][/{variant}]]
```

**Review by tejlmand — COMMENTED** [review]
LGTM, a +1 from here.

Two minor observations where I think things could be misinterpreted for newcomers.

File: doc/hardware/porting/board_porting.rst (hunk 14 above)
Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst:80]: Well spotted.

File: doc/hardware/porting/board_porting.rst (hunk 15 above)
Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst:159]: Indeed

**Review by tejlmand — COMMENTED** [review]
LGTM, a +1 from here.

**Review by carlescufi — DISMISSED** [review]
+1 from @tejlmand

**Review by ajarmouni-st — DISMISSED** [review]

**Review by gmarull — DISMISSED** [review]

File: doc/glossary.rst [hunk 16]
```diff
@@ -85,12 +128,30 @@ Glossary of Terms
       integrated circuit that are not in use.
 
    SoC
-      `System on a chip`_
+      A `System on a chip`_, that is, an integrated circuit that contains at
+      least one :term:`CPU cluster` (in turn with at least one :term:`CPU core`),
+      as well as peripherals and memory.
+
+   SoC family
+      One or more :term:`SoCs <SoC>` or :term:`SoC series` that share enough
```
Comment (nordicjm) [inline doc/glossary.rst:136]: Should there be a term for extended board?

File: doc/hardware/porting/board_porting.rst [hunk 17]
```diff
@@ -65,112 +65,128 @@ complete conversion reference.
...
+   |                                            +--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   |                                            | nrf5340/cpunet           | nRF5340     | nRF53              | Nordic nRF         | Arm Cortex-M33 | Arm Cortex-M         |
+   +--------------------------------------------+--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8ml8/a53             | i.MX8M Plus | i.MX8M             | NXP i.MX           | Arm Cortex-A53 | Arm64                |
+   |                                            +--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   |                                            | mimx8ml8/m7              | i.MX8M Plus | i.MX8M             | NXP i.MX           | Arm Cortex-M7  | Arm Cortex-M         |
```
Comment (nordicjm) [inline doc/hardware/porting/board_porting.rst:114]: architecture isn't really right, M4 would be armv7, M33 would be armv8

File: doc/hardware/porting/board_porting.rst [hunk 18]
```diff
@@ -65,112 +65,128 @@ complete conversion reference.
...
-A board may be a physical piece of hardware or an emulated board.
-Furthermore a board may contain one or multiple SoCs. Also, each SoC may contain
-one or multiple CPU clusters. A CPU cluster refers to a group of CPU cores.
-Only CPU cores of same architecture can be in the same cluster. In the case
-where a physical SoC considers a CPU cluster to contain CPU cores of different
-architectures then those must be modelled as multiple clusters, where all CPU
-cores within a cluster is having the same architecture.
-It is possible to have only a single CPU core within a CPU cluster.
+Zephyr's hardware support hierarchy has the following levels, from most to least
+specific:
```
Comment (henrikbrixandersen) [inline doc/hardware/porting/board_porting.rst:74]: Not sure I agree that the list goes from "most to least specific", but it's not a blocker.

File: doc/hardware/porting/board_porting.rst (hunk 17 above)
Comment (henrikbrixandersen) [inline doc/hardware/porting/board_porting.rst:114]: "Arm Cortex-M" isn't really the architecture in Zephyr terminology, though - is it?

Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst:114]: yes, good point. I will fix that.

**Review by henrikbrixandersen — DISMISSED** [review]

File: doc/hardware/porting/board_porting.rst (hunk 17 above)
Comment (carlescufi) [reply doc/hardware/porting/board_porting.rst:114]: Yep, see the comment above

**Review by kartben — CHANGES_REQUESTED** [review]
quick NACK on the size of the PNG file, while I do a more thorough review. 
209 KB does't sound right at all for such a simple file. Options: running it through optipng, look at WEBP format, or even straight SVG (which would likely also preserve the drawio metadata to allow for editing should it be needed)

See below (note that setting a smaller width (here 800px, but 500 might be even more appropriate since that's your target in the actual documentation page) would be better done directly when exporting from the tool you used, as it would minimize bluriness.

```
cwebp board-terminology.png -resize 800 0 -o out.webp                                           INT ✘  11:20:44  
Saving file 'out.webp'
File:      board-terminology.png
Dimension: 800 x 239
Output:    13920 bytes Y-U-V-All-PSNR 41.66 43.50 45.59   42.40 dB
           (0.58 bpp)
...
```

File: doc/hardware/porting/board_porting.rst [hunk 19]
```diff
@@ -65,112 +65,128 @@ complete conversion reference.
...
-.. _hw_support_hierarchy:
+which can be extended to:
 
-Boards, SoCs, etc.
-******************
+.. code-block:: console
 
-Zephyr's hardware support hierarchy has these layers, from most to least
-specific:
+   {board name}[@{revision}][/{SoC}[/{CPU cluster}][/{variant}]]
```
Comment (kartben) [inline doc/hardware/porting/board_porting.rst:159]: The combination of `[ ]`s and `{ }`s makes this really hard to parse visually IMO
Please consider the following:

```
Formally this can also be seen as
:samp:`{board name}[@{revision}][/{board qualifiers}]`, which can be extended to
:samp:`{board name}[@{revision}][/{SoC}[/{CPU cluster}][/{variant}]]`.
```

image

File: doc/hardware/porting/board_porting.rst [hunk 20]
```diff
@@ -65,112 +65,128 @@ complete conversion reference.
...
-         CPUs, be these in different SOCs or in a SOC with multiple AMP CPU
-         clusters.
-- SoC: the exact system on a chip the board's CPU is part of
-- SoC series: a smaller group of tightly related SoCs
-- SoC family: a wider group of SoCs with similar characteristics
-- CPU Cluster: a cluster of one or more CPU cores.
-- CPU core: a particular CPU in an architecture
-- Architecture: an instruction set architecture
+If a board contains only one single-core SoC, then the SoC can be omitted from
+the board target. This implies that, if the board does not define any board
```
Comment (kartben) [inline doc/hardware/porting/board_porting.rst:162]:
```suggestion
the board target. This implies that if the board does not define any board
```

**Review by kartben — COMMENTED** [review]
looking good, thanks for taking care of the image size @carlescufi. Another additional comment for improved readability, please.

**Review by gmarull — APPROVED** [review]

File: doc/glossary.rst (hunk 16 above)
Comment (carlescufi) [reply doc/glossary.rst:136]: Not sure what you mean by that?

**Review by kartben — APPROVED** [review]

File: doc/glossary.rst (hunk 16 above)
Comment (nordicjm) [reply doc/glossary.rst:136]: This feature was added some time ago https://github.com/zephyrproject-rtos/zephyr/pull/57390/files might make sense to have the term added to the glossary

**Review by carlescufi — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

File: doc/glossary.rst (hunk 16 above)
Comment (carlescufi) [reply doc/glossary.rst:136]: ooh right. The glossary needs tons of updates, among them this one :)  I will improve the glossary soon with another PR

**Review by iuliana-prodan — APPROVED** [review]
